### ISSUE TITLE
Method 1 and Method 2

### ISSUE THREAD

**Issue by JChazzer** [mikealmond/hq-trivia-assistant#63]
Hi,

Having used this bot a few times, it seems clear that Method 2 is far more accurate than Method 1, which would be fine as they are both displayed. However, Method 2 often appears after the question has ended, or very late in the timer. 

It would seem to me that the bot would be far more accurate if Method 2 came first, and was wondering how easily this could be done? I tried simply the swapping the two bits of code around but was thrown an error.

Thanks!

**Comment by JChazzer** [comment]
Also, what is the difference between the two methods?

**Comment by tombets2424** [comment]
I would guess that they either compile the results differently or they use different ways of obtaining the "correct" answer (like different search engines).

**Comment by manofsteel2017** [comment]
i agree. method 2 is better but slower. got eliminated just now because method 1 can't get it right lol
all good we can't solely rely on this 
hopefully better bots come out soon
